internal/learn/the-community/qanda.md

```diff
@@ -0,0 +1,98 @@
+
+**Accounting**
+In discussions, there was always the suggestion that you could combine individual micro-transactions and then pay monthly, for example as with a subscription. If one would try this as a company it would require the creation of an accounting system that is able to handle millions of data-Tx.
+
+These need to be…
+
+1. evaluated (because they can all have completely different prices).
+2. assigned (thousands of data tx would have to be assigned to the respective user).
+3. calculated (for each user a final invoice would have to be made and possibly be charged with their data-tex if you are a user yourself).
+4. check (that the payments have actually been made).
+
+That would result in…
+
+1. an immense staff body
+2. a high risk of errors
+3. a high demand for IT and office equipment for the personnel in question
+4. thus high and hardly calculable costs.
+
+Conclusion: Completely unnecessary effort of resources and personnel, which the protocol provides free of charge and fully automated with the help of IOTA Smart Contracts, IOTA streams etc.
+
+Btw.: All companies have to document and tax value transactions. So all data transactions have to be recorded in blockchains for accounting purposes. With IOTA, not only the fees for data transactions are eliminated but also the associated accounting including Tx storage (up to 10 years) for the majority of all transactions.
+
+**Chain of events**
+Only with a system in which data transactions can be linked directly to value transactions the complete event chain of the data trade can be correctly traced. The IOTA token reflects the (potential) value generated by the data transaction. Therefore, it would make no sense at all if only the IOTA data transactions were used but the payment of this data takes place outside the IOTA protocol.
+
+Payment in the same system is very advantageous for all parties involved, for example, if it is done in a digital decentralized marketplace. Let’s take a look at the current process for placing an order. As a rule, an order always starts a series of events that are based on each other. This could look like this: Order > Verification > Confirmation > Payment > Check receipt of payment > Processing and shipping. This process can of course be solved individually or partially automated but basically this chain of events is always the same.
+
+The IOTA technology with the digital marketplaces enables a free, secure and trustless linking of unknown business partners. The technology offers a very cost effective or even free possibility to link the chain of events of an order in a single system at the correct time. If one were to outsource individual events, e.g. external payment without IOTA, it would be an additional effort to enter this individual event back into the event chain of the marketplace at the correct time. This would cause additional costs compared to the pure IOTA solution and the IOTA solution would therefore have competitive advantages.
```

Comment:
```suggestion
The IOTA technology with the digital marketplaces enables a free, secure and trustless linking of unknown business partners. The technology offers a very cost-effective or even free possibility to link the chain of events of an order in a single system at the correct time. If one were to outsource individual events, e.g., external payment without IOTA, it would be an additional effort to enter this individual event back into the event chain of the marketplace at the correct time. This would cause additional costs compared to the pure IOTA solution, and the IOTA solution would therefore have competitive advantages.
```